### PR TITLE
C359592 Verify updating Email in Bulk edit

### PR DESCRIPTION
**Pull request by MaksimTerekhov**
https://foliotest.testrail.io/index.php?/cases/view/359592

### CODE REVIEW

File: cypress/support/fragments/bulk-edit/bulk-edit-actions.js [hunk 1]
```diff
@@ -99,6 +99,12 @@ export default {
     cy.expect(HTML('Replace with').is({ disabled: true }));
   },
 
+  replaceEmail(oldEmailDomain, newEmailDomain) {
+    getBulkEditSelectType().select('Email');
+    cy.get('[class^=textField]').first().type(oldEmailDomain);
```
Comment (IhorBohdan) [inline cypress/support/fragments/bulk-edit/bulk-edit-actions.js:104]: If interactor 'TextField' not working move to const

**Review by IhorBohdan — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

File: cypress/support/fragments/bulk-edit/bulk-edit-actions.js (hunk 1 above)
Comment (MaksimTerekhov) [reply cypress/support/fragments/bulk-edit/bulk-edit-actions.js]: thx, fixed

**Comment by penfriend** [comment]
Do you remove your file userBarcodes_239.4173868307128402 after test executed ?

**Comment by MaksimTerekhov** [comment]
> Do you remove your file userBarcodes_239.4173868307128402 after test executed ?

thx, fixed

**Pull request merged**